class.jetpack.php

```diff
@@ -22,6 +22,8 @@
 	Flag for "activating" the plugin on sites where the activation hook never fired (auto-installs)
 */
 
+require_once( JETPACK__PLUGIN_DIR . '_inc/lib/class.media.php' );
```

Comment:
Would it make sense to use https://github.com/Automattic/jetpack/blob/764a699120ca6795dac3d264207713516200ee7b/require-lib.php#L3 instead?